filescppgen.py

```diff
@@ -16,25 +16,25 @@ def usage():
 
 
 try:
-    files_version = sys.argv[1]
```

Comment:
This variable is still used below - see https://travis-ci.org/openmicroscopy/ome-release/builds/201208041